src/map_io/s2map.cc

```diff
@@ -795,7 +795,9 @@ void S2MapLoader::load_s2mf(Widelands::EditorGameBase& egbase) {
 			Widelands::MapIndex const index = Widelands::Map::get_index(location, mapwidth);
 			c = bobs[index];
 			std::string bobname;
-			if (buildings[index] == 0x78) {
+			// TODO(Nordfriese): The codes for the stones are used for multiple things depending
```

Comment:
checking the site provided in a comment above https://settlers2.net/documentation/world-map-file-format-wldswd/ 
I found out that this is not controlled by the Buildable field code (misleadingly called buildings here).    
stones and trees and landscape objects are controlled by section 6 which should be renamed and used for this task here. 
the logic of section 6 is shortly 
1. 0x80(128) = HQ --> value in block 5 is player number
2. 0xc4(196) = tree group 1 --> value is a random animation state (1 of 8) of a tree in group 1  
3. 0xc5(197) = tree group 2 --> value is a random animation state (1 of 8) of a tree in group 2  
4. 0xc6(198) = tree group 3 --> value is a random animation state (1 of 8) of a tree in group 3
5. 0xc8(200) = landscape object --> value is the code of a landscape object
6. 0xcc(204) = Stones Type 1 --> values is the amount (1-6) of the first stones type
7. 0xcd(205) = Stones Type 2 --> values is the amount (1-6) of the second stones type